Atlas/backend/authentication/models.py

```diff
@@ -0,0 +1,58 @@
+from django.contrib.auth.models import AbstractBaseUser, BaseUserManager
+from django.db import models
+
+
+class AccountManager(BaseUserManager):
+    def create_user(self, email, password=None, **kwargs):
+        # Ensure that an email address is set
+        if not email:
+            raise ValueError('Users must have a valid e-mail address')
```

Comment:
Also; with these credentials, I'm getting an exception from server because of the password being less than 7 characters:
```json
{ 
  "email": "yigitozkavci8@gmail.com",
  "username": "yigitozkavci",
  "password": "123123",
  "confirm_password": "123123"
}
```
 We need to return 400 status, not 500 with internal server error in this case.